MdePkg/Include/Base.h

```diff
@@ -206,6 +206,27 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 
 #define PACKED
 
+// Omit Function from Stack Overflow Protection.

```

Comment:
Add a few more details to make sure it is clear this is an attribute for a function declaration and document when it should or should not be applied to a function